gnomad_qc/v3/annotations/generate_sv_hists.py

```diff
@@ -103,41 +100,39 @@ def get_sample_age(sv_list: hl.Table) -> hl.Table:
         ),
         release=sample_meta[sv_list.s].release,
     )
-    logger.info(
-        "%i out of %i samples in the SV sample list have a defined age.",
-        sv_list.filter(hl.is_defined(sv_list.age)).count(),
-        sv_list.count(),
-    )
-    logger.info(
-        "%i out of %i samples in the SV sample list are in the core release",
-        sv_list.filter(sv_list.release).count(),
-        sv_list.count(),
-    )
     return sv_list
 
 
 def get_sex_and_autosome_mt() -> hl.MatrixTable:
-    """Read in and union autosome and sex chromosome VCFs."""
+    """
+    Read in and union autosome and sex chromosome VCFs.
+
+    :return: MatrixTable containing calls from autosome and sex chromosome VCFs.
+    """
     logger.info("Importing VCFs...")
+    # NOTE: The sex chromosome VCFs have an extra "PAR" field that needs to
```

Comment:
same question about `# noqa` here

Reply:
This is under 88 characters so I think it should be fine